assets/css/admin/style.scss

```diff
@@ -93,12 +93,33 @@ body.post-type-download #screen-meta {
 	padding: 0 20px;
 }
 
+#edd-header-branding {
+	vertical-align: middle;
+}
+
 #edd-header img {
-	display: block;
 	max-width: 300px;
 	margin: 0 0 0 20px;
 }
 
+.edd-header-page-title-wrap {
+	margin-top: 7px;
+	font-size: 22px;
+	line-height: 28px;
+	vertical-align: top;
+	display: inline;
```

Comment:
```suggestion
```

Reply:
This and the vertical-align can be removed because the parent element is using flexbox. `align-items: center;` should be added to that. Additionally, this rule could be simplified to this:

```css
.edd-header-page-title-wrap {
    margin-top: -5px;
    font-size: 22px;
    margin-right: auto;
    padding-left: 7px;
}
```